src/sagemaker/workflow/utilities.py

```diff
@@ -458,6 +465,8 @@ def trim_request_dict(request_dict, job_key, config):
         request_dict.pop(job_key, None)  # safely return null in case of KeyError
     else:
         if job_key in request_dict:
+            if request_dict[job_key] is None:
```

Comment:
what if request_dict[job_key] is an empty string? is that a valid case?

Reply:
Shouldnt be. Its dependent on all the class constructors resolving that field as a default to None. Should we add guardrails for that as well ? That would have to be manually inputted by the CX. aka `model-name=""`